### DISCUSSION TITLE
Quais ferramentas utilizar para monitoramento e funil?

### DISCUSSION THREAD

**Discussion by evelew** [frontendbr/forum#1530] [Perguntas e Respostas]
Oie, no lugar em que trabalho estamos tendo dificuldades para monitorar nossa aplicação, por isso gostaria de saber quais ferramentas vocês utilizam para fazer esse trabalho. 

E também estamos tendo problemas com os funis, como a aplicação tem muitos fluxos precisamos de algo bem maleável para entender as ações dos clientes. 

Se vocês puderem me falar o que utilizam pra esses dois casos eu agradeço demais. 
Valeuuu :)

**Comment by kavalcante** [comment]
E ai, beleza?

Aqui na empresa utilizamos as seguintes ferramentas:

- Monitoramento: [Sentry](https://sentry.io/welcome/), [Rollbar](https://rollbar.com/), Apollo Engine e [Pingdom](https://www.pingdom.com/).
- Funis: [Amplitude](https://amplitude.com/), mas tbm pode ser utilizado o [Google Analytics](https://analytics.google.com).

**Comment by felipefialho** [comment]
Tentamos o Amplitude, não gostamos tanto. Já o Google Analytics, acho fantástico pra qualquer tipo de produto.
